microsoft/MixedReality-GraphicsTools-Unity
# Automated CI Pipeline

## Overview
An automated ADO pipeline to push to the MR GraphicsTool package to MRTK NPM feed

## Changes
- Fixes: https://github.com/microsoft/MixedReality-GraphicsTools-Unity/issues/68.

## Verification (Before Merge)
- Check that NPM feed for the test version pushed via the pipeline

## Post-merge tasks
- [ ] Remove current branch from exclude list.
- [ ] Update pipeline to point to main
- [ ] Create a new release tag for 0.4.14 

## Verification (After Merge)
- Check that the latest release is pushed to the MRTK NPM feed.
- Check that the pipeline only runs for release tags.
